css/trackers.scss

```diff
@@ -121,13 +122,18 @@ body {
 ul.js-menu-item-list {
     list-style-type: none;
     padding-left:14px;
+    margin-top:0;
 }
 
 // .js-site-item
 ul.js-menu-item-list li {
     padding-top:17px;
 }
 
+ul.js-menu-item-list li.js-site-co:first-child {
+    padding-top:0 !important;
```

Comment:
Generally `!important` is an anti-pattern. Is there something you are trying to override here? Am assuming its just the rule above it. The rule should work without the !important. Let me know if there is something else going on. 

One other thing -- we need to separate the style rules used for css and the style rules used for js. So the `js-site-co` should only be for working with within javascript logic, and you would apply a class without the `js-` prefix for styling.

We also need to start using scss nesting instead of raw css:
http://sass-lang.com/documentation/file.SASS_REFERENCE.html#nested_rules